### PR TITLE
Switch vocabularyTerm lookup from search to find

### PR DESCRIPTION
**Pull request by markquintontulloch**
Search for 'Annotation notes' was also returning 'Disease note
types' terms, presumably due to the 'note' part.

### CODE REVIEW

**Review by oblodgett — COMMENTED** [review]
Does the search endpoint not work?

**Comment by markquintontulloch** [comment]
@oblodgett - It retrieves the 'Disease annotation notes' terms along with the 'Annotation types' terms - presumably due to the shared 'note'.

**Comment by oblodgett** [comment]
Then something is wrong that shouldn't happen... this is used in many places so we would need to look into this issue more.

File: src/main/cliapp/src/service/useControlledVocabularyService.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export function useControlledVocabularyService(termType) {
 
   useQuery(['terms', termType],
     () => {
-      return searchService.search("vocabularyterm", 15, 0, null, { "vocabFilter": { "vocabulary.name": { "queryString": termType } } })
+      return searchService.find("vocabularyterm", 15, 0, {"vocabulary.name" : termType } )
```
Comment (oblodgett) [inline src/main/cliapp/src/service/useControlledVocabularyService.js:31]: This is probably better in the long run to pull the list directly from the the database vs ES.

**Review by oblodgett — APPROVED** [review]

**Pull request merged**